insights/parsers/ethtool.py

```diff
@@ -816,31 +816,16 @@ def parse_content(self, content):
                 except:
                     raise ParseException('Ethtool unable to parse content', line)
 
-        self._supported_link_modes = []
+        self.supported_link_modes = []
         if 'Supported link modes' in self.data:
             for pair in self.data['Supported link modes']:
-                self._supported_link_modes += pair.split()
+                self.supported_link_modes += pair.split()
 
-        self._advertised_link_modes = []
+        self.advertised_link_modes = []
         if 'Advertised link modes' in self.data:
             for pair in self.data['Advertised link modes']:
-                self._advertised_link_modes += pair.split()
+                self.advertised_link_modes += pair.split()
 
-        self._supported_ports = []
+        self.supported_ports = []
```

Comment:
After doing that, you need to add these `Attributes` to the docstring explicitly.